linetools/lists/linelist.py

```diff
@@ -778,13 +845,15 @@ def available_transitions(self, wvlims, n_max_tuple=None, min_strength=0.):
             return None
 
         # create auxiliary Table
+        pdb.set_trace()
         unique = Table()
         unique['name'] = transition_name
         unique['rel_strength'] = strength
+        '''
 
```

Comment:
I'm not sure the sorting is done right now... originally, we want to sort by strenght but considering only the strongest available transition in a given ion...

Reply:
am pretty sure it is preserved.  I had to fuss a bit
to match the tests (and the ordering in those).